deploy/packages/bbp-packages.yaml

```diff
@@ -35,6 +35,7 @@ packages:
       - py-entity-management@1.0.12
       - py-jupyter-notebook@4.2.3
       - brion@3.0.0 +python
+      - ultraliser
```

Comment:
Your intuition is right. This should actually be in `gnu-stable` if it does not depend on Python. See the very bottom of this file.